frontend/src/components/SignupForm.jsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from "react";
+import axios from 'axios';
+
+const SignupForm = () => {
+  const [username, setUsername] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newUser = { username, email, password };
+    console.log(newUser);
+    axios.post('/api/signup', newUser)
+      .then(response => {
+        console.log(response.data);
```

Comment:
does the back end redirect the user to the '/' or home page after signing up?  Right now the front-end console logs with no redirects on successful signup.

Reply:
Should it redirect to the home page with them logged in after successful signup?